data/mods/Magiclysm/monsters/Orcs.json

```diff
@@ -56,18 +62,19 @@
         { "item": "armguard_scrap", "prob": 40 },
         { "item": "cuirass_scrap", "prob": 40 },
         { "item": "longbow", "prob": 100 },
-        { "item": "sheath", "contents-item": "knife_combat", "prob": 50 }
+        { "item": "sheath", "contents-item": "knife_combat", "prob": 50 },
+        { "item": "quiver_large", "contents-group": "quiver_orc_archer", "prob": 100 }
       ]
     },
-    "starting_ammo": { "arrow_wood_heavy": 75 },
+    "starting_ammo": { "arrow_wood_heavy": 60 },
     "extend": {
       "special_attacks": [
         {
           "type": "gun",
           "cooldown": 10,
           "move_cost": 93,
           "gun_type": "bow",
```

Comment:
```suggestion
          "gun_type": "longbow",
```

Reply:
~~Sorry for not explaining the rationale behind these changes.~~ This takes an item id with type gun or just `gun`, `bow` trows some generic factory errors A.K.A. this item does not exist and is the main reason they can't fire, 
It works
![acurateorc](https://user-images.githubusercontent.com/58050969/113624247-fe0a2980-9624-11eb-872a-2bc9ae8b465d.png)

Reply:
Palm to head... oops! Thank you!